hooks/useZoraPremint.tsx

```diff
@@ -15,44 +15,95 @@ const useZoraPremint = () => {
   const { sendTransaction: sendTxByWallet } = useWalletSendTransaction()
   const { connectWallet } = usePrivy()
 
+  const IS_TESTNET = process.env.NEXT_PUBLIC_TESTNET === "true"
```

Comment:
IS_TESTNET should already be exported from `consts.tsx`
- if it's not there already, can you move this declaration to the `consts.tsx` file?

Reply:
Added this comment changes to the PR.